### PR TITLE
Website: Add ember-router-scroll

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

If merged, this PR adds `ember-router-scroll` as a dependency, to handle "scroll to top" behavior.

Note: if I open a new tab and visit a hash URL, it's being removed. I don't think it's this addon, since we were seeing it before, I think it's something in one of the controllers. I haven't been able to pinpoint exactly what is causing this behavior, but I'll keep working on it. 

In the meantime, this addon is a step in the right direction for our app.

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

**Comment by Dhaulagiri** [comment]
> if I open a new tab and visit a hash URL, it's being removed

I see that too 😢 . Not sure where or when that broke 😬 

This PR fixes _most_ of the issues outlined in the original bug, but still doesn't work in the instance of the navigating to the the accessibility link in the footer. Any idea why that might be?

**Comment by Dhaulagiri** [comment]
> Not sure where or when that broke 😬

Looking back and checking out old code it seems possible this never actually worked 😫

**Comment by MelSumner** [comment]
> > Not sure where or when that broke 😬
> 
> Looking back and checking out old code it seems possible this never actually worked 😫

Ok try it now, I think I had the config incorrect. I *think* it's working locally.

File: website/config/environment.js [hunk 1]
```diff
@@ -6,6 +6,11 @@ module.exports = function (environment) {
     environment,
     rootURL: '/',
     locationType: 'history',
+    historySupportMiddleware: true,
+    routerScroll: {
+      targetElement: '#main',
```
Comment (MelSumner) [inline website/config/environment.js:11]: TODO try to target higher in the page

Comment (MelSumner) [reply website/config/environment.js:11]: TODO Make sure to try not targeting anything to see if it works better.

File: website/config/environment.js [hunk 2]
```diff
@@ -38,8 +41,8 @@ module.exports = function (environment) {
   if (environment === 'development') {
     // ENV.APP.LOG_RESOLVER = true;
     // ENV.APP.LOG_ACTIVE_GENERATION = true;
-    // ENV.APP.LOG_TRANSITIONS = true;
-    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
+    ENV.APP.LOG_TRANSITIONS = true;
+    ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
```
Comment (didoo) [inline website/config/environment.js:45]: do we need these uncommented? there is  already a lot of noise in the console when building the app, and we may risk to lose important messages there

Comment (Dhaulagiri) [reply website/config/environment.js:45]: @MelSumner can we get these commented out again so we can get this PR finished?

File: website/config/environment.js [hunk 3]
```diff
@@ -38,8 +41,8 @@ module.exports = function (environment) {
   if (environment === 'development') {
     // ENV.APP.LOG_RESOLVER = true;
     // ENV.APP.LOG_ACTIVE_GENERATION = true;
-    // ENV.APP.LOG_TRANSITIONS = true;
-    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
+    ENV.APP.LOG_TRANSITIONS = true;
```
Comment (MelSumner) [inline website/config/environment.js:44]:
```suggestion
    // ENV.APP.LOG_TRANSITIONS = true;
```

File: website/config/environment.js [hunk 4]
```diff
@@ -39,7 +42,7 @@ module.exports = function (environment) {
     // ENV.APP.LOG_RESOLVER = true;
     // ENV.APP.LOG_ACTIVE_GENERATION = true;
     // ENV.APP.LOG_TRANSITIONS = true;
-    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
+    ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
```
Comment (MelSumner) [inline website/config/environment.js:45]:
```suggestion
    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
```

**Review by Dhaulagiri — APPROVED** [review]

**Pull request merged**